Lib/timeit.py

```diff
@@ -260,6 +266,9 @@ def main(args=None, *, _wrap_timer=None):
     if args is None:
         args = sys.argv[1:]
     import getopt
+    import _colorize
+    colorize_errors = _colorize.can_colorize()
```

Comment:
```suggestion
    import _colorize
    import getopt
    colorize = _colorize.can_colorize()
```

Sorry, `colorize` could be better as we may want to actually use it later for even better reporting (like we could colorize the results themselves).

We should also import `_colorize` alphabetically.

Reply:
You can also move those imports before `if args is None`, but keep `colorize = ...` after the if block.